WalletWasabi/BitcoinCore/RpcClientBase.cs

```diff
@@ -0,0 +1,166 @@
+using NBitcoin;
+using NBitcoin.RPC;
+using System;
+using System.Threading.Tasks;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.BitcoinCore
+{
+	public class RpcClientBase : IRPCClient
```

Comment:
I am not sure the name is the best description for this class.
This should be a first hand citizen of wasabi and the current RpcClient probably used internally to the class and not known outside of its context.
Other options are just go for `RpcService`  or maybe just to differ `BitcoinClient`?

Reply:
`RpcClientBase` is not the best name and `BitcoinClient` is a better, I agree. However, I would prefer to have that discussion in a following PR.